features/FEATURE_BLE/targets/TARGET_CORDIO/stack/mbed_os_adaptation/wsf_os.c

```diff
@@ -49,7 +49,7 @@ WSF_CT_ASSERT(sizeof(uint32_t) == 4);
 
 /* maximum number of event handlers per task */
 #ifndef WSF_MAX_HANDLERS
-#define WSF_MAX_HANDLERS      9
```

Comment:
This configuration change is specific to the NRF52840, and is not meant for this PR. 
Please overide that macro in `targets.json`.